src/framework/filterControl/forms/FilterForm.jsx

```diff
@@ -13,6 +13,8 @@ import DateRangeFilterForm from './DateRangeFilterForm.jsx';
 import InputFilterForm from './InputFilterForm.jsx';
 import MixedTypeValueFilterForm from './MixedTypeValueFilterForm.jsx';
 import MultipleSelectFilterForm from './MultipleSelectFilterForm.jsx';
+import SearchableMultipleSelectFilterForm
```

Comment:
why new line here?

Reply:
the eslint config has a strict 80 char limit and this line goes over that